src/electionguard/elgamal.py

```diff
@@ -24,6 +24,7 @@
 ElGamalPublicKey = ElementModP
 
 _BLOCK_SIZE = 32
+_ELGAMAL_DATA_SIZE = 512
```

Comment:
I feel like this can and should use the Padded Data Size that already exists. These should be using the same number but more importantly ARE the same number. They are just set at different points. 

https://github.com/microsoft/electionguard-python/blob/5b73f75cc54f63837bf7cd6328adff208e6dca62/src/electionguard/serialize.py#L51

Reply:
Steve is correct, but I can see how they're tightly related and there should probably be an underlying 512 constant, so that's what it does currently